examples/spatial_tiny_yolo.py

```diff
@@ -118,18 +118,15 @@
     counter = 0
     fps = 0
     color = (255, 255, 255)
+    dispScaleFactor = calculateDispScaleFactor(device)
 
     while True:
         inPreview = previewQueue.get()
         inDet = detectionNNQueue.get()
-        depth = depthQueue.get()
+        inDepth = depthQueue.get()
 
         frame = inPreview.getCvFrame()
-        depthFrame = depth.getFrame()
-        depthFrameColor = cv2.normalize(depthFrame, None, 255, 0, cv2.NORM_INF, cv2.CV_8UC1)
-        depthFrameColor = cv2.equalizeHist(depthFrameColor)
-        depthFrameColor = cv2.applyColorMap(depthFrameColor, cv2.COLORMAP_JET)
-
+        depthFrame = parseDepthPacket(inDepth, dispScaleFactor, stereo.getMaxDisparity())
```

Comment:
You should target develop with this PR.
`stereo.getMaxDisparity()` is deprecated there.